boards/shields/coverage_support/Kconfig.shield

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config SHIELD_COVERAGE_SUPPORT
+	def_bool $(shields_list_contains,coverage_support)
+	select HAS_COVERAGE_SUPPORT
```

Comment:
You probably want to use snippets, https://docs.zephyrproject.org/latest/build/snippets/index.html

Reply:
Thanks for the suggestion, I did not know that possibility.
However, when I started to use snippets I found an easier way, just using KCONFIGs directly.
I will stick to that simple solution until more settings do not need to be adjusted to get coverage.